react-components/hello-world.md

```diff
@@ -95,8 +91,13 @@ When working with React, or other autoloading programs, it's a good idea to disa
 
 ### !end-callout
 
-![A web browser displaying the default React application. It consists of the React logo, a note to "Edit src/App.js and save to reload," and a link to additional resources for learning about React.](../assets/react-components_hello-world-landing.png)  
-_Fig. The default React application created by `npx create-react-app`_
+### !callout-warning
+
+## Server Not Reloading? Stop and Restart It
+
+If the auto-reload feature does not work (we're editing code, but not seeing any changes in the browser) then it might be necessary to completely stop your server with ctrl-C in the terminal window where it's running, then restarting it again with `npm run dev`.
```

Comment:
```suggestion
If the auto-reload feature does not work (we're editing code, but not seeing any changes in the browser) then it might be necessary to completely stop the server with ctrl-C in the terminal window where it's running, then restarting it again with `npm run dev`.
```